components/topbar/user/src/index.js

```diff
@@ -246,14 +246,15 @@ class TopbarUser extends Component {
             </div>
           </div>
         </div>
-        <div className="sui-TopbarUser-ctaButton">
+        <div className='sui-TopbarUser-ctaButton'>
           <AtomButton
             link
             href={navCtaUrl}
             title={navCtaText}
-            leftIcon={<navCTA.icon svgClass="sui-TopbarUser-ctaButtonIcon" />}
-            size="small"
-            type="accent"
+            leftIcon={<navCTA.icon svgClass='sui-TopbarUser-ctaButtonIcon' />}
+            size='small'
+            type='accent'
+            onClick={navCtaOnClick}
```

Comment:
The prop name should be more specific IMO.
onCTAClick, onLoginClick, etc

Reply:
👍